chaosk8s/deployment/actions.py

```diff
@@ -0,0 +1,81 @@
+import json
+import os.path
+import yaml
+
+from chaoslib.exceptions import ActivityFailed
+from chaoslib.types import Secrets
+from kubernetes import client
+from logzero import logger
+from kubernetes.client.rest import ApiException
+
+from chaosk8s import create_k8s_api_client
+
+__all__ = ["create_deployment", "delete_deployment"]
+
+
+def create_deployment(spec_path: str, ns: str = "default",
+                      secrets: Secrets = None):
+    """
+    Create a deployment described by the deployment config, which must be the
+    path to the JSON or YAML representation of the deployment.
+    """
+    api = create_k8s_api_client(secrets)
+
+    with open(spec_path) as f:
+        p, ext = os.path.splitext(spec_path)
+        if ext == '.json':
+            deployment = json.loads(f.read())
+        elif ext in ['.yml', '.yaml']:
+            deployment = yaml.load(f.read())
+        else:
+            raise ActivityFailed(
+                "cannot process {path}".format(path=spec_path))
+
+    v1 = client.AppsV1beta1Api(api)
+    resp = v1.create_namespaced_deployment(ns, body=deployment)
+    return resp
```

Comment:
should we return the serialized version of the response?

usually the response is logged as-is in the chaostoolkit.log file and I'm not sure this format will be readable?

Reply:
I looked in other location, and create functions doesn't have return 
I've just deleted it

If returning response is wanted, we can create another PR by doing it everywhere